modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -157,33 +156,6 @@ class ElementWiseLayer : public Func::Layer
 
     virtual Ptr<BackendNode> tryAttach(const Ptr<BackendNode>& node) CV_OVERRIDE
```

Comment:
Seems to me method `tryAttach` was used for Halide backend only. Correct me if I'm wrong. If so, let's remove them all.

Reply:
It looks like the same to me. Lets remove them all then.